lib/led-matrix.cc

```diff
@@ -183,16 +184,45 @@ uint8_t RGBMatrix::brightness() {
   return brightness_;
 }
 
+void RGBMatrix::SetTransformer(CanvasTransformer *transformer) {
+  transformer_ = transformer;
```

Comment:
Will add the linked transformer as standard transformer. If there is no other transformer inside the linked transformer then the input canvas will be returned.